talisker/flask.py

```diff
@@ -52,36 +52,46 @@ def sentry(app, dsn=None, transport=None, **kwargs):
 def add_view_name(response):
     try:
         endpoint = flask.request.endpoint
-        module = flask.current_app.view_functions[endpoint].__module__
+        name = endpoint
+        if flask.current_app:
+            module = flask.current_app.view_functions[endpoint].__module__
```

Comment:
When trying to access an endpoint that doesn't exist I get an Exception that is printed in the console:

```bash
2018-05-01 08:53:48.281Z ERROR talisker.flask "could not get view name from flask request" request_id="5007d585-1c42-49e0-a8f8-68fc7b4a8f0a"
Traceback (most recent call last):
  File "PATH/talisker/talisker/flask.py", line 57, in add_view_name
    module = flask.current_app.view_functions[endpoint].__module__
KeyError
```

Reply:
Ugh, that's ugly.

While its not directly related to this issue, I've pushed a fix to this branch for this issue, as it's a fairly small fix.

In the process, I discovered that raven.contrib.flask 5.27.0 error when there is no view, so will convert any 404 into a 500! So I moved the minimum supported version to the 5.27.1 bugfix release in the process.